exercises/parallel-letter-frequency/parallel_letter_frequency_test.go

```diff
@@ -44,3 +44,25 @@ Gave proof through the night that our flag was still there;
 O say does that star-spangled banner yet wave,
 O'er the land of the free and the home of the brave?`
 )
+
+func BenchmarkSequentialFrequency(b *testing.B) {
+	b.StopTimer()
+	b.StartTimer()
+
+	for i := 0; i < b.N; i++ {
+		Frequency(euro + dutch + us)
+	}
+
+	b.StopTimer()
```

Comment:
Looks like StartTimer() and StopTimer() are only needed if you want to do stop the clock while doing some expensive setup (https://golang.org/pkg/testing/#B.StartTimer)